src/main.ts

```diff
@@ -12,22 +13,22 @@ interface RequestedMonitoringEntry {
 }
 
 interface SummaryState {
-	secondaries?: Record<string, SummaryState>,
-	timeMean: number,
-	timeStd: number,
-	memMean: number,
-	memStd: number,
-	cpuMean: number,
-	cpuStd: number,
-	eventLoopLagMean: number,
-	eventLoopLagStd: number,
-	actionsPerSecondMean: number,
-	actionsPerSecondStd: number
+	secondaries?: Record<string, SummaryState>;
```

Comment:
AFAIK semicolons after interface properties are a convention. At least `Prettier` enforces it.